SamMou.Api.Graph/QueryObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using GraphQL;
+using GraphQL.Types;
+using SamMou.Api.Models;
+using SamMou.Api.Services.Interface;
+
+namespace SamMou.Api.GraphQL
+{
+    public class QueryObject : ObjectGraphType<object>
+    {
+        public QueryObject(IWeatherForecastService service)
+        {
+            try
+            {
+                Name = "Queries";
+                Description = "The base query for all the entities in our object graph.";
+
+                FieldAsync<WeatherForecastObject, WeatherForecast>(
+                    "singleweatherforecast",
+                    "Gets weatherforecast",
+                    new QueryArguments(
+                    new QueryArgument<NonNullGraphType<IdGraphType>>
+                    {
+                        Name = "id",
+                        Description = "The unique GUID of the movie."
```

Comment:
here you name it weather. we can keep this service for weather right? then when we work on movie, we create another query for it?

Reply:
I copy this from tutorial and forgot to rename it😅